### DISCUSSION TITLE
Week 3 Thinking Cap: UX Design

### DISCUSSION THREAD

**Discussion by yohman** [yohman/21S-DH151#26] [Assignments]
What is UX design?

Read the following article ["Introduction to UXD"](https://medium.com/beakerandflint/an-introduction-user-experience-design-2a7f8167bf03) and write a short reaction piece to it.

Additionally, submit one or two questions for our guest speaker next week, [Alesha Unpingco](https://ace.alumni.ucla.edu/alesha-unpingco/).

**Comment by alicelu170** [comment]
**Reflections on UXD**
I think it is fascinating to see the amount of work that goes into designing and creating products that intentially appear simple and are often taken for granted. I was also surprised by how many levels UX designers work on, from evaluating need all the way to designing the final product. It makes sense that UXD essentially provides a voice for the user, since they often do not have the chance to work closely with a company. However, I understand that this overview of UXD is very preliminary and I am interested in hearing more about the nuances of this process.

**Questions**
Many facets of the UXD process reminded me of community-centered design, such as in areas of architecture and urban planning. In these instances, community-centered design attempts to create a paradigm shift by bringing community members and stakeholders (the users) directly into design processes. While I understand that UXD tries to plan for the user's experience, do you think a community- and user- centered approach to product design that relies less on UXD as a mediator between users and companies is possible?

With UXD perhaps becoming more common and well-known, have you seen a shift in the background that your coworkers come from? That is, what areas of study or experiences do those involved in UXD have?

Inclusivity is important for ensuring the user experience is positive across large populations. In your experience, are your teams diverse in background, race, SES, ability, gender, sexuality, etc.? How have you felt this diversity (or lack thereof) affect the product design process?


https://github.com/alicelu170/DH151/blob/main/Week%203/W3%20Thinking%20Cap.md

**Comment by MaryJaneHowland** [comment]
**Reflection:**

Before reading the article and watching the video, I was under the impression that UX Design was a process spearheaded solely by website developers and coders. Now, I have learned that UX Design is a collaborative process that actually involves many people with different backgrounds. Visual designers, for example, can be apart of the UX Design process and can work with coders to achieve desired visual aesthetics of a website or app. I have also realized that UX is not always just one specific phase of the design process of an app/site, but it is a way of thinking, and therefore can and should be incorporated into all the stages of building digital products and sites.

**Questions for Alesha:**

(1) What strategies do you implement when thinking about the UX design for new projects? Basically, what is your process?

(2) What advice do you have for someone who is interested in the UX field but isn't a strong coder or doesn't have much experience in the field?

**Comment by justisg** [comment]
UX Design Reflection
Fall quarter this year, I took a course on UX/UI Design within the Digital Humanities. Prior to that course, I had absolutely no experience in the field, but through the work, I was able to learn more. I specifically was made aware of the importance of user testing and getting feedback from many individuals to create the best possible end product. It is an interesting field, as the article states, because of the ways it intersects web building and technology and creativity and design. It is a typically unseen role, however, it is imperative, especially for building a website. In the article, I was very interested in looking at the UX pyramid, which I have not seen before. It seems to be a very useful resource for prioritization and focus within this work. Especially given the importance of functionality on mobile devices, focus on how a user will interact with a product, site, or application is significant for its success. Within the video, I enjoyed hearing about how UX designers deal with problems in their work and the way their designs evolve throughout the entire process.

Guest Speaker Questions: Alesha Unpingco
Can you explain the best way you have found to prioritize your work in this field?
How have you incorporated accessibility into your work?

**Comment by derekwang99** [comment]
**Reaction to "Introduction to UXD"**
A particular quote that I liked from the article is that “there is no need to reinvent the wheel each time” and that designers can and should use available internet resources and common design practices to promote consistency. I think this concept was hard to accept for me because throughout my entire education, I was taught to seek out originality. However, in the case of UX, originality may not always be the best decision if it drastically differs from current conventions. I’m currently taking DH110 Intro to UX Design and appreciate how this article is validating what I am learning (for example, advocating the use of heuristic evaluation).

**Reaction to "What is user experience design?"**
I like how this video simplifies the process of UX design into four stages: understand the user, defining a problem, designing a solution, and making the product fun to use. The previous article mentioned how sometimes it is difficult for him to explain his job to family members or friends, but this video gives a brief description of what he does. I also like how the video allows the viewer to realize the invisible impact of good user design.

**Questions For Alesha:**
_Question 1:_ I’m currently in the process of deviating away from a math background and am instead pivoting towards UX design. I was wondering if you have any advice as to what I should do over the summer to make the most out of it.

_Question 2_: How was the transition from working at an advertising agency to working at a large tech corporation? What is your favorite project that you have worked on over your career?

**Comment by anncong** [comment]
**Reaction**
I think the UX Pyramid is a great breakdown and tool for me to use as a refresher. I haven't learned about the UX pyramid, but I've read similar articles about it. My favorite phase is getting to know more about the user because I like how it can pave different ways for the product to be more suited to its user-base.

**Questions for Guest Speaker**

1. Do I absolutely need a higher degree to work as a UX researcher? My "dream" job would be to work as a UX researcher.

2. I have been doing personal projects for around year now, and I just recently started participating in UX-thons to get to know the community and to get more hands on experience working with other people , but I am not sure if I am getting the experience needed to put on my portfolio and to get hired when I graduate. Are there any tips you can give me?

**Comment by michelle14lee** [comment]
**Reaction Piece**
I thought the introduction video gave a good overview of the UX Design process and its applications. I had no idea that there were so many specialized roles even within the divison of UX like user researcher, interaction designer, and visual designer! From the article, it's also interesting to see that UX is a people-first role, where they prioritize how to make products or features more intuitive and enjoyable for their users. I think UX is often overlooked because it exists in so many everyday tools and technologies that people take it for granted. The article and video gave me a fresh perspective on the role, and that it's a very important and intentional part of the intersection between business, people, and technology.

**Questions for Alesha Unpingco**

1. What important skills did you pick up in this career that you didn't expect to need when you first entered the field?
2. From your experience, how has UX differed at the different companies/industries you've worked in (startups vs big/emerging tech)?

**Comment by hanarama** [comment]
**Reflections**
I feel like good UXD is like any other piece of infrastructure: it's only noticeable when it's bad. When UX is well designed, you don't even think about how easy and convenient everything is.

Ralph mentions how important it is to engage with users from the very beginning when designing UX. I think it's particularly important to engage with disabled users. Disabled users will be able to point out flaws that are inconvenient to most users, but present complete barriers to them. For example, many of the websites used to book vaccine appointments are difficult for low-vision users to read and don't function easily with screen readers. Working with vision-impaired users during design would have made the websites more useable by all users.

I have a friend who works in UXD that fights to include disabled voices in the design process. They have told me that whatever the effort they have to expend in order to make a design fully accessible in the beginning is a lot lower than the effort it takes to make the design accessible later on. Kind of like how it's easier to design a building that is accessible to begin with than it is to add in ramps and elevators later after the building is already finished.

**Questions for Alesha**

1. Are there any unique challenges in UXD for AR/VR compared to other things? I'm not able to use VR technology really because it makes me feel motion sick and the goggles rarely accommodate my glasses.
2. How do you incorporate accessibility into your designs?

**Comment by JuliaWood1** [comment]
### Short Reaction Piece:
There is so much thought that goes into every single product!! User Experience Design is the iterative process behind all of the products we use everyday. This design process is undertaken by a group of creative minds who always have the users need's at the forefront of their mind. However, before a designer can even attempt to understand the user, they need to identify viable solutions to users' pain points. While many people may have the same pain points or problems, there are many different types of users, and that is why a UX Design team needs to be a diverse, creative group of people so they can truly empathize with a wide array of people. There a lot of main design heuristics that designers abide by, in order to ensure that people intuitively understand how to navigate and interact with their product. We all know, if we can't figure out a product in 15 seconds, we won't be likely to use that product again. Also designers need to passionate about their solution, if they this product or feature is not going to make an important change in someone's life, is it good design? No. Throughout this whole process UX Designers need to constantly ask questions about the user, like where are they using this product, how are they using, and WHY are they using it? Not only does a UX designer need to ask questions, but they need to answer them by watching users interact with their product. From observing the user, one can then improve their product to prepare it for the general public. UX Design looks like a very fun and creative career path! 

### Question for Alesha Unpingco:
1. What is your guiding principle for designing an interactive user experience?

**Comment by jmtanenbaum** [comment]
I was very interested to learn about all the layers of UX design. I always thought of UX as an artistic role, something mainly focused on creating attractive beautiful content. However, the research aspect of UX design really appeals to me as an oral historian and past researcher. However, the communication problems between the many individuals on UX teams, customers, developers, and other individuals seem difficult to overcome. I spent a couple of years as a freelancer and like working with one or two people, so this sounds a bit daunting to me. I also wish the piece would have touched on accessibility, which is a crucial aspect of design and an ethical imperative. 

My question for Alesha is: What is it like working with so many team members on large projects? Is the interpersonal aspect difficult and what strategies would you suggest for best dealing with that?

**Comment by zonaliao** [comment]
**UX Design Reaction:**

Learning about UX Design was really interesting because I didn't know too much about it before, but through watching the video and reading about User Experience I can see how many of the products that we use daily all have some extent of user experience research put into it. They do a lot of research and study human behaviors in order to understand what would be the best way to go about creating a successful product for customer use. There is a lot more to it than just plain design, so I think it is cool to read about how each tier in the pyramid functions as a multistep process to get to the finalized product. It seems like usability and function plays a big part in the process for creating the flow of the user's experience because people want to use something that is easy to understand while still improving their lives in some way.

**Questions:**

1. As a UX Designer, do you do any work that also overlaps with UI Design and how do they function together?
2. What made you get into UX Design and what do you wish you knew before starting this career path?

**Comment by adrianaromero819** [comment]
**UX Design Reaction** 

I have a close friend whose dad works as a UX designer, and the way he always explained it to me is similar to what is described in the article: designing digital experiences based on the behavior and motivations of the website user. Whenever I enter a website that is difficult to navigate and not very user-friendly or a website that is aesthetically pleasing and has features that make it easy to navigate, I think about UX design. It is interesting to think about all of the components that UX designers consider when designing a website, which is evident in the UX Pyramid. I only really think about how functional, convenient, and pleasurable a website is to visit and use. I never thought about aspects of reliability, ease of use, and meaningfulness; yet, I know that all components of the pyramid greatly impact my user experience. The amount of thought and collaboration that goes into user experience design was fascinating to learn about, and I enjoyed how the video incorporated the research process that goes into design. User experience design teams are largely responsible for our app and website experience without us even realizing it. I think it is all really awesome. 

**Question for Alesha Unpingco**

1. You stated you did not graduate from a Design/Media Arts program and I see that you got your degree in English Literature. How has your degree been useful and insightful in your career in UX design? 
2. Are there ever times that a project or business you are working with has clashed with your passion for service and social good? How do you address such challenges, balancing work and personal beliefs? Or how does your voice make a difference in the different work you partake in?

**Comment by ivanma9** [comment]
Reaction: 
Coming into the article, I had a brief understanding of UI and UX because it is talked about in computer science a lot, but the talks are very surface level. Understanding the pyramid of necessities for user experience, it was very insightful to see the checklist of requirements for a user designer. As a web developer, I often set subtasks for certain features that go in a similar order as the pyramid, but seeing the breakdown of functionality, reliability, and usability in terms of development is really good. I have always learned to make it work first and then make it pretty and that is kind of what it says here in the pyramid. However, functionality and design can be split up into many parts. This where the user research comes in to make the ideal type of experience for the product being selled.

Questions:
1. Are changes often made mid-development process or before during "research" stage?
2. How do you conduct "user research"?

**Comment by ryanorlando** [comment]
**Reaction:**
Before reading the article and watching the video, I had a very basic understanding of UX, but not much more than that. As someone who has studied psychology in the past, I was very interested to see that the hierarchy of UX principles is based on Maslow's Hierarchy of Human Needs. This is an interesting concept, because that psychological idea is rooted more in survival and fulfillment, and UX provides more of a brief experience (however long someone is using the designed tool). However, it is easy to see where these concepts overlap, because the goal of designing a tool or platform is to get users to come back and use it again, so it makes sense to root the design in a concept that encourages reliance and enjoyment. In that regard, I was also very interested to see how much thought goes into the enjoyment experience of users, as that is not really something I would normally think about while using a simple app or website. After thinking about it, I realized that in this case, maybe enjoyment is typically closer to "not overwhelmingly frustrated", and not having the user think about enjoying the platform at all could be considered a success. Most likely, it depends on the initial goals of the tool, and the constraints of the developers.

**Questions:**

1. When given different UX design projects, are the goals always clear in terms of the relationship between functionality and enjoyment, or are you given more creative freedom to balance the two how you see fit?
2. How has the field of UX changed in the time you have worked in it? Has new research and technology created any big changes for you and your design process?

**Comment by yjwsb233** [comment]
**Reaction:**
I hear about UX design all the time but the article and the video gave me a systematic understanding of what it is about. Like pointed out by the video, UX research and design involve complex layers of work that start from observation, survey, and research, and move on to design and concept realization. Both the article and the video prove that UX design is a field where people from diverse disciplines come together and work towards the same goal. What I found interesting from reading the article is the importance of asking the right questions. While most of the time we start a project with ambitions and good intentions, sometimes we might get too focused on a single aspect that we risk jeopardizing the bigger picture. Therefore, I totally agree with Alesha that we should not be afraid of feedback and instead should constantly seek them. 

**Questions:**
1. What are some specific things that you do in your current job?
2. How do you think user experience manifests itself in the organization of a university?

**Comment by kaisanwatanabe** [comment]
I enjoyed the way the article broke down UX into a hierarchical pyramid. It made it easier to understand why different applications I've used made have fallen short, as well as some I've created myself. I also appreciated the article mentioning general solutions to approaching a UX problem, such as using existing frameworks or evaluating your own design choices using heuristic evaluation. I currently do a lot of work creating wireframes for apps, and so it was helpful to know about the common practices UX designers go through so that I may interface it into my own workflow. I think it's imperative that this type of thinking is used across the board, and that collaboration occurs between different teams within a project (designers, developers, marketers). Doing so will likely speed up the overall workflow because each team is reporting design flaws that they have noticed, and in turn give room for people to come up with creative solutions without the fear of failure being too impactful to the project.

Questions:
1. When working as a UX designer, how much intention did you put into exploring new interaction patterns versus retaining current viable ones? (how do you make sure you aren't reinventing the wheel while trying to come up with something new?)
2. As someone who enjoys both coding and graphic design, I'm curious to know how the realms of development and design work together at top tech companies. What are good examples of collaboration you have seen between the two, and what are some areas in which designers and developers can do better to work more smoothly?

**Comment by kimjee8955** [comment]
Reflection: 
As someone who has very little knowledge on design, it was interesting to learn more about UX Designers do. Reading the article and watching the video made me realize that I've taken most of my online experience for granted. There is actually a lot of research, creativity, and collaboration that goes into designing a website. One common theme I noticed across the article and the video is that a UX designer should be able to ask insightful questions about the product and the user such as 'who is using the product?', 'why will the users use the solution?', etc. The process sounds like the steps in scientific method: asking a research question, formulating hypothesis, coming up with wireframes and prototypes to test, then repeating the process if the test doesn't work out. 

Questions:

1. Where do you go for your UX Design inspirations?
2. How do you determine when your project has succeeded? (I realized that I keep fixing minute details when I design a website. How do you know when you are a good place to stop?)

**Comment by LynnT2** [comment]
Reflection: I am working to become a UX designer, so the article and the video both gave me a refresher on the basics of UX design. They reminded me why I am pursuing this career, and the prospects that I have in it.  I found it interesting that the article itself didn't go into depth about the design cycle, and the importance of iteration in a product. However, I enjoyed how the "UX pyramid" was explained. Reflecting on my own work and using the pyramid as reference, I feel that I often neglect the "meaningful" and "pleasurable" parts of the framework to focus on the functional and useful. I was reminded that I should create enjoyable experiences as well as functional ones, and by extension, tap into the joy from the work that I do. 

Questions:
1. As a person who is trying to enter the UX field from an Economic background, I'm always looking for advice from UX designers who came from backgrounds that don't necessarily apply to their current career. How did you get into UX design as an English Lit major, and do you have any advice on how people from non-tech backgrounds could enter the world of tech through UX design?
2. Could you talk about your favorite project that you've been able to work on as a UX designer?

**Comment by jomh17** [comment]
Reflection:
UX is so interesting because as the author said, it can be hard to define. So many things go into a project that the user isn't aware of. I really enjoyed the pyramid of needs and considerations that go into the UX design process. It was an interesting way to represent the thought process the designer goes through when fixing a website. The only issue with this methodology is that is is vary hard to quantify or judge. Whether or not these qualities are fulfilled can change from person to person and is subjective.  I think the main thing the article was missing is the user involvement. For a website to be successful, you should have user testing and feedback to make sure that any problems are fixed and the considerations are noticed by the user.  

Questions:
1. What is the most important thing we need to know about UX that makes it seem like we are pros in an interview?
2. How do you make sure that all your projects don't look the same when you are using the same considerations for every project? 
3. How do you balance the user and the customer you are fixing the website for? What if they are wrong?

**Comment by natgrace** [comment]
Reflection: Although I knew a lot of work goes behind creating a website, it's the little details you don't really realise you need to prioritise until you make one yourself...and then suddenly, it's the most important part of the website. UX design is always spoken about as if all it does is add the pretty parts of a website, but the article and video really breaks down the process and the meticulousness that goes behind every decision. As someone who had difficulty working on making her own story map user-friendly, I can't imagine the back and forth discussions that take place behind a website with a whole team behind it. It's even scarier to think about social media sites and how changing things up will give you live and immediate feedback from its millions of users everyday. I also enjoyed the article breaking it down into a pyramid because it really helps prioritise certain things over others yet, showing that at the end of the day, the user experience is more than just knowing where the home and X buttons are.

Questions: 
1) What are some things you feel people tend to forget the most when they make a website? 
2) Do you think there is a benefit of using services (wix, wordpress) to make websites (over starting from scratch)?
3) Would you mind sharing one interesting "characteristic" you've added to a site before that you found really cool? Thank you!

**Comment by danielliuzy** [comment]
I have coded apps as personal side projects, where there isn't really any requirements or rules to follow. As someone without any training in UX design or other similar experiences (and lacking in talent in that department), it was always a pain for me when it came to the UX design parts. Prior to working on my own apps, I thought UX design is something anyone can pick up on easily. Watching the video and reading the article really opened my eyes to how much there is in the UX design world and made me appreciate UX designers more, as their work is just as, if not more important as engineers. Reflecting upon my past works, most of the time I would put too much emphasis on the "Pleasurable" level before more basic levels, such as "Reliable" and "Usable" have been fully satisfied. Although I do not plan to be a UX designer in the future, I think learning the basics of UX design will be useful for communicating with UX designers, as what they do are important parts of the process of building applications.

Questions:
1. How is UX design different for AR/VR compared to traditional screens? 
2. What are some frustrations you have experienced when communicating with engineers?

**Comment by lisalou36** [comment]
**Reaction:**
I never thought much about UX Design before taking a Library Carpentries course focused on it a few months ago. Essentially the course was highlighting how designers are instrumental in making public resources, such as bus stop benches, accessible or prohibitive to the public. I find the UX Design field fascinating because it's one of those jobs where if you do it well, then users don't even think about the person who made those decisions (the pinnacle rule of good infrastructure). I appreciate the author's take that a UX Designer's job is to be the "customer advocate," emphasizing this point further by positioning "Meaningfulness" at the top of the UX Design pyramid. As someone looking to become a librarian or archivist with a focus on instruction and accessibility upon graduating, I look to always incorporate such an emphasis on human-centered accessibility (so it looks as though I can learn much from this field!).

**Question for Alesha:**
1. I'm really interested in data ethics, especially given the enormous increase in power and influence we have seen in tech companies over the past two decades. I'm curious how conversations about data ethics are incorporated into the UX Design field?
2. I'm also really interested in tactics you use to get out of creative ruts. I love making art, but I feel like my creativity and imagination is blocked most of the time. When (if) you experience these creative blocks, what do you do to help yourself get inspired again?

**Comment by alekhyavittalam** [comment]
**Reaction:**
I thought the article was very well written, especially for someone who does not know much about UX Design. I had taken a class on UI/UX at UCLA and found it very interesting how we have to think about 'why' the user would find the product useful. The pyramid in the article gives a good introduction to the basics of UX Design and what we as beginners to UI/UX need to think about when designing our own products. I'm really interested in learning more about this field!

**Questions:**
1. How did you get interested in the UX Design field?
2. What resources do you recommend for someone interested in starting a career in UX Design with little to no experience?

**Comment by hoz-map** [comment]
**Reflection**

Until reading the article and watching the video, I thought I understood what UX design was. However, the article made me notice I did not know anything about UX design. Specifically, it was eye-opening to know how many different types of experts are involved in creating a design and how many steps a designer needs to take to produce one. Also, the process was much more scientific than I thought. I am looking forward to learning more about the details of those processes and the works that UX designers do from Alesha.


**Questions**

1. What does your typical day look like as a UX designer?
2. How long do you usually take to finish one project?
3. Is there any recommendation for websites or anything we can learn about UX design? (Also, do you have any advice for students who just become interested in the field of UX design?)

**Comment by alexthegreat1** [comment]
### Reaction

Reading the linked article and watching the video on UX Design was very enlightening. The article's breakdown of the UX Pyramid was extremely helpful because it helped me understand what order UX design tends to go in and what is most important. Breaking it all down into 6 simpler parts makes it easier to digest and focus on each part at a time. I also loved the explained characteristics for each division of the pyramid in practical terms. I will definitely be applying these characteristics and questions to our project for this class and any projects I work on in the future.

As for the video, it really helped me understand how integrated UX Design is in the creation and manufacturing of the product. There are many more steps to it than I originally thought, such as research, drawing, wireframes, prototyping, etc. It seems like a very involved process and the end product of what UX Designers can do is very impressive.

### Questions

1. Is there a specific platform/goal/idea you've wanted to do/execute that you haven't had the chance to yet?
2. What platform (mobile, desktop websites, AR, etc.) do you most enjoy doing UX design for? Why?

**Comment by ainsleighdouglas** [comment]
Thinking Cap

Response
Over the past few digital humanities courses I feel like I have gotten a stronger hold on the creative process in UX/UI design, like heuristic research. However, an interesting new tool that I learned through the reading was the UX pyramid. I think the interesting idea behind this was what is necessary and what is "additional". I wouldn't go as far as to say the top of the pyramid is unnecassary but it is the line that makes a different between pro and novice. 

 Questions
What is the collaboration process like on a team with multiple disciplines? how roles like visual designer and user researcher overlap during the creation process?

https://github.com/ainsleighdouglas/DH-151/blob/main/Week3/thinkingcap.md

**Comment by sebdills** [comment]
I had the opportunity to learn about UI/UX design in DH150 and got to design a high-fidelity prototype for an app idea I had. The whole process as well as this article showed me how common poor design is. It made me realize that if I'm struggling to do something on a website etc, it most likely isn't my fault as a user but an indication that the UX is poorly designed. There is no reason for websites to be difficult to navigate. I also realized how much goes into the process of web design (and design in general).

Questions:
What helped you self teach a in order to get into UI/UX, what was your approach and any tips?
What do you like about your job?

**Comment by kaicolorado** [comment]
<b>Reflection</b>
I find the field of user experience design very interesting. The process of using a webpage or other product is not just a means of reaching the desired output, but an experience in itself. It is essential, for any good product, to have a user friendly design that minimizes confusion and possible error. In doing this, there exists a balance between extreme functionality and easily understood simplicity. I believe that deciding on which features are to be implemented and how to implement these features is the most important part of user design.

<b>Questions</b>
How does desktop UX design differ from mobile? How have touchscreens changed UX design?
How closely do you work with other departments throughout the design process?

**Comment by lisa58** [comment]
**Reaction Piece**
Prior to taking Digital Humanities for my technical breadth, I didn't know anything about UX Design so I found this article very informative for me because it gave me more insight about the field itself and gave me more of an idea as to whether or not I wanted a career in UX Design in the future. It made me realize how much I take certain websites for granted. It's much easier to notice a website that is hard to navigate through as opposed to one that is user friendly.

**Questions for Alesha Unpingco**
How different is the working environment at Google compared to RPA? What was the switch from RPA to the Google like?
What were some of the Super Bowl campaigns you worked on?

**Comment by kmcinerny** [comment]
While I am opinionated about aesthetics and pay attention to them in digital spheres, I never knew about the process of UX. The testing and tweaking required in UX make sense (even I have spent hours tweaking map aesthetics). However, I still didn’t previously consider how many iterations / steps there are. I think about how crucial UX design is, especially when it comes to Apple products. The aesthetic and user-friendly UX of Apple is probably its biggest draw (considering that Apple falls short in other areas- like life of their products and functionality, when compared to PCs).

**Questions**
- What was your path from an English Literature major to the UX profession? What drew you to UX?
- What is the difference between AR and VR?
- Can you tell us about some of the AR/VR projects you've worked on at Google?

**Comment by frida-barragan** [comment]
Reaction to _Introduction to UXD_ :
I enjoyed reading this article because I have always been very interested in understanding how aesthetics and functionality relate to user experience (I think it is fascinating how the layout/design of a web page can affect the motivations of the users). I had some previous knowledge about UX design but it was great reading more about the UX Pyramid and all the elements/questions that arise when creating and designing. As a heavily visual person, I can now appreciate even more all the effort that goes into creating aesthetically pleasing websites while making them reliable and functional. One of my favorite websites is Peter Tarka's (https://www.petertarka.com). I think Tarka's website is simple but emphasizes the pieces so well and truly highlights his skills.

Questions for Alesha Unpingco:
1. What websites do you think are good examples of great UX design?
2. How did you overcome the challenges of learning UX design? Also, do you have any resources you could share for self-learners?

**Comment by ec028** [comment]
I think the thing that stood out most to me was the following quote, "balancing business goals with technology constraints". I think this is something that is important to keep in mind for our own projects; for me personally, trying to keep goals realistic while still maintaining some sort of ideal visualization in mind is something I tend to have trouble with. For instance, I often want to make layouts a certain way for projects, but time constraints and unexpected factors (breaking of the website, components not working) tend to keep me from achieving those goals. I also liked how practical UX research and the UX creation process was; often it seems more like an artistic endeavor than a scientific one, but the fact that there is data collection, testing, and the repetition of this process shows that UX research is much like other fields of research -- it's just there's also a more immediately tangible component to it in the form of a finished site/layout.

**Questions for Alesha Unpingco**
1. Do you have any tips for transitioning from English to UX design? What skills should we be learning?
2. What were some skills that were translatable from your major to your eventual field?
3. How is the teamwork environment like in a given project/challenge? What is a typical workday like? 
4. Do you find yourself working in an interdisciplinary manner? Does the work that you do feel artistic/creative, or does it feel more methodical?

**Comment by eliaranita1** [comment]
Reaction:
Before reading these I kind of thought of UX/UI as just fancy terms for graphic design and that they were mostly about aesthetics. It was interesting to see how much thought actually goes into things that sometimes, once done, just seem so obvious and natural. The Norman doors were a great example of bad design.

Q1:
How did you get into the field of UX/UI?

Q2:
What is an example of a design idea of yours that you’re particularly proud of?
